### ISSUE TITLE
python-jl not finding python script file when called via unit file (systemd)

### ISSUE THREAD

**Issue by AssafLS** [JuliaPy/pyjulia#426]
I have some python code that calls a Julia script that works fine on windows 10.
When I tried to run it on Ubuntu I got the "Your Python interpreter is statically linked to libpython" message, using compiled_modules=False option in the Julia constructor did not work and resulted in the same error message.
Running my python script using python-jl worked fine from the command line.
However, when I try to run this as a service using a unit-file located in etc/systemd/system and starting the service I get a file not found message from the python-jl proccess:

my-service.service
   Loaded: loaded (/etc/systemd/system/my-service.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Wed 2020-10-07 11:12:29 UTC; 7s ago
  Process: 7643 ExecStart=/home/someuser/.local/bin/python-jl /home/someuser/services/pp.py (code=exited, status=1/FAILURE)
 Main PID: 7643 (code=exited, status=1/FAILURE)

Oct 07 11:12:29 SolverSim1 python-jl[7643]:     os.execvp(cmd[0], cmd)
Oct 07 11:12:29 SolverSim1 python-jl[7643]:   File "/usr/lib/python3.8/os.py", line 568, in execvp
Oct 07 11:12:29 SolverSim1 python-jl[7643]:     _execvpe(file, args)
Oct 07 11:12:29 SolverSim1 python-jl[7643]:   File "/usr/lib/python3.8/os.py", line 610, in _execvpe
Oct 07 11:12:29 SolverSim1 python-jl[7643]:     raise last_exc
Oct 07 11:12:29 SolverSim1 python-jl[7643]:   File "/usr/lib/python3.8/os.py", line 601, in _execvpe
Oct 07 11:12:29 SolverSim1 python-jl[7643]:     exec_func(fullname, *argrest)
Oct 07 11:12:29 SolverSim1 python-jl[7643]: FileNotFoundError: [Errno 2] No such file or directory
Oct 07 11:12:29 SolverSim1 systemd[1]: my-service.service: Main process exited, code=exited, status=1/FAILURE
Oct 07 11:12:29 SolverSim1 systemd[1]: my-service.service: Failed with result 'exit-code'.

If I use the same unit file but with the python3 path instead of the python-jl the file is found but I get back to my original problem with the message: "Your Python interpreter is statically linked to libpython"

Unit file (my_service.service):
[Unit]
Requires=network.target remote-fs.target
After=network.target remote-fs.target

[Service]
User=someuser
Group=someuser
WorkingDirectory=/home/someuser/services
ExecStart=/home/someuser/.local/bin/python-jl /home/someuser/services/pp.py
Restart=on-abnormal

[Install]
WantedBy=multi-user.target

**Comment by tkf** [comment]
I guess you don't have `julia` in the `PATH` inside the service. If so, I think you can fix it by tweaking `PATH` of the service or passing `--julia=FULL/PATH/TO/julia` to `python-jl`.

**Comment by AssafLS** [comment]
Thanks @tkf for responding so quickly!
Indeed setting the PATH environment variable inside the .service file did the job correctly.
The code I added was:
`
...
[Service]
...
Environment="PATH=$PATH:/home/someuser/julia-1.5.2/bin"
...`

BTW, I also tried using the python-jl --julia <julia-path> workaround, and it only only worked for simple python scripts that don't use the julia python module, this kinda misses the purpose of using python-jl as a workaround for running PyJulia on Linux machines.
